Dockerfile

```diff
@@ -1,31 +1,31 @@
 # Dockerfile for Sphinx SE
 # https://hub.docker.com/_/alpine/
-FROM alpine:3.14
+FROM --platform=linux/amd64 alpine:3.18
```

Comment:
What is the point of setting the platform explicitly? You can always build for desired platforms using `docker buildx` for example.

Having the platform set here will decrease performance on arm, because I guess it will always produce amd64 build.

Reply:
Sphinx offers no ARM binaries, your Dockerfile explicitly downloads linux-amd64-musl.
So any attempt to build with an alpine ARM image will fail:
```
> [5/7] RUN indexer -v:
#0 0.292 qemu-x86_64: Could not open '/lib/ld-musl-x86_64.so.1': No such file or directory
```
If you agree that the `docker run --platform linux/amd64` in Makefile make sense, I would like to make Dockerfile robust also, no matter what host wants to build it.
Change this line to `FROM --platform=linux/arm64 alpine:3.18` and try it yourself ;)

Reply:
Oh, that makes sense, thank you.